docs/developer-guide/ui-extensions.md

```diff
@@ -33,32 +33,45 @@ The resource tab extension should be registered using the `extensionsAPI.registe
 registerResourceExtension(component: ExtensionComponent, group: string, kind: string, tabTitle: string)
 ```
 
-* `component: ExtensionComponent` is a React component that receives the following properties:
+- `component: ExtensionComponent` is a React component that receives the following properties:
 
-    * application: Application - Argo CD Application resource;
-    * resource: State - the kubernetes resource object;
-    * tree: ApplicationTree - includes list of all resources that comprise the application;
+  - application: Application - Argo CD Application resource;
+  - resource: State - the kubernetes resource object;
+  - tree: ApplicationTree - includes list of all resources that comprise the application;
 
-    See properties interfaces in [models.ts](https://github.com/argoproj/argo-cd/blob/master/ui/src/app/shared/models.ts)
+  See properties interfaces in [models.ts](https://github.com/argoproj/argo-cd/blob/master/ui/src/app/shared/models.ts)
 
-* `group: string` - the glob expression that matches the group of the resource; note: use globstar (`**`) to match all groups including empty string;
-* `kind: string` - the glob expression that matches the kind of the resource;
-* `tabTitle: string` - the extension tab title.
-* `opts: Object` - additional options:
-  * `icon: string` - the class name the represents the icon from the [https://fontawesome.com/](https://fontawesome.com/) library (e.g. 'fa-calendar-alt');
+- `group: string` - the glob expression that matches the group of the resource; note: use globstar (`**`) to match all groups including empty string;
+- `kind: string` - the glob expression that matches the kind of the resource;
+- `tabTitle: string` - the extension tab title.
+- `opts: Object` - additional options:
+  - `icon: string` - the class name the represents the icon from the [https://fontawesome.com/](https://fontawesome.com/) library (e.g. 'fa-calendar-alt');
 
 Below is an example of a resource tab extension:
 
 ```javascript
 ((window) => {
-    const component = () => {
-        return React.createElement( 'div', {}, 'Hello World' );
-    };
-    window.extensionsAPI.registerResourceExtension(component, '*', '*', 'Nice extension');
-})(window)
+  const component = () => {
+    return React.createElement("div", {}, "Hello World");
+  };
+  window.extensionsAPI.registerResourceExtension(
+    component,
+    "*",
+    "*",
+    "Nice extension"
+  );
+})(window);
+```
+
+## Sidebar Extensions
```

Comment:
I think Sidebar Extension is a bit ambiguous. It also might mean components rendered in a sidebar. One option is to rename it to `System Level Extension` ?